.github/workflows/attach-release-assets.yaml

```diff
@@ -110,7 +110,7 @@ jobs:
             }
 
       - name: Find matching files in build artifacts
-        uses: tj-actions/glob@v17
+        uses: tj-actions/glob@v22
```

Comment:
verified no breaking change for this workflow by going through the release notes: https://github.com/tj-actions/glob/releases